docs/odata-csdl-json/odata-csdl-json.md

```diff
@@ -4301,56 +4301,113 @@ Addresses/-1/Street
-targeting the directly enclosing type.
-
-For annotations targeting a structural or navigation property of an
-entity type or complex type, the path is evaluated starting at the
-*outermost* entity type or complex type named in the target of the
-annotation, i.e. an empty path resolves to the outermost type, and the
-first segment of a non-empty path MUST be a structural or navigation
-property of the outermost type, a [type cast](#TypeCast), or a [term
-cast](#TermCast).
-
-For annotations embedded within or targeting an action, action import,
-function, function import, parameter, or return type, the first segment
-of the path MUST be a parameter name or `$ReturnType`.
+Annotations MAY be embedded within their target, or specified separately,
+e.g. as part of a different schema, and specify a path to their target model
+element. The latter situation is referred to as *targeting* in the remainder of
+this section.
+
+If the value of an annotation is expressed dynamically with a path
+expression, the path evaluation rules for this expression depend upon the model
+element by which the annotation is hosted. In this subsection, an annotation
+is said to be *hosted* by its target, unless that target is another annotation
+or a collection, record or property value of another annotation, in which case
+it is hosted by the same model element that also hosts that other annotation.
+
+For annotations hosted by an entity container, the path is evaluated starting
+at the entity container, i.e. an empty path resolves to the entity container,
+and non-empty paths MUST start with a segment identifying a container child
+(entity set, function import, action import, or singleton). The subsequent
+segments follow the rules for path expressions targeting the corresponding
+child element.
+
+For annotations hosted by an entity set or a singleton, the path is evaluated
+starting at the entity set or singleton, i.e. an empty path resolves to the
+entity set or singleton, and non-empty paths MUST follow the rules for
+annotations targeting the declared entity type of the entity set or singleton.
+
+For annotations hosted by an entity type or complex type, the path is
+evaluated starting at the type, i.e. an empty path resolves to the type, and
+the first segment of a non-empty path MUST be a structural or navigation
+property of the type, a [type cast](#TypeCast), or a [term cast](#TermCast).
+
+For annotations hosted by an action, action import, function, function
+import, parameter, or return type, the first segment of the path MUST be the
+name of a parameter of the action or function or `$ReturnType`.
+
+For annotations hosted by a structural or navigation property, the path
+evaluation rules additionally depend upon how the annotation target is
+specified, as follows:
+
+- If the annotation uses targeting and the target path starts with
+  an entity container, or the annotation is directly or indirectly embedded in such an
```

Comment:
Can a path really start with an entity container? Aren't all paths technically start with a schema namespace or alias?

Reply:
Target paths of an `<Annotations>` element can start with an entity container.